Makefile.am

```diff
@@ -207,14 +216,30 @@ gresources += data/moltres/moltres.gresource
 pkgdata_DATA += $(gresources)
 
 # # # MAIN LIBRARY # # #
+BUILT_SOURCES = \
+	lib/eosknowledgeprivate/ekn-resources.h \
+	lib/eosknowledgeprivate/ekn-resources.c \
+	$(NULL)
+
+ekn_resources_deps = $(shell $(GLIB_COMPILE_RESOURCES) --generate-dependencies --sourcedir=lib/eosknowledgeprivate/ lib/eosknowledgeprivate/ekn-resources.gresource.xml)
+lib/eosknowledgeprivate/ekn-resources.h: $(ekn_resources_deps)
+lib/eosknowledgeprivate/ekn-resources.c: $(ekn_resources_deps)
```

Comment:
Because %.gresource.deps generates dependencies for .gresource target and I need the deps for .c and .h files